src/CalcManager/UnitConverter.cpp

```diff
@@ -191,6 +191,11 @@ void UnitConverter::SwitchActive(const wstring& newValue)
     }
 }
 
+bool UnitConversionManager::UnitConverter::IsSwitchedActive() const
+{
+    return m_switchedActive;
+}
+
```

Comment:
Before, after user switches active and then changes the currency unit, the input may be blocked.
This is because the unit changed to has less fraction digits than the previous unit.

However, as the new value will always replace the old value after switched active, user should be able to input regardless of the old value. 

So, I have to add this method so that `ViewModel` can decide whether to block the input per `switchedActive`.

Reply:
I think we'd better add some real currency data (only abbr. matters, the rates can be fake) in UI Tests.
Otherwise, we cannot easily test the case mentioned here.

Reply:
@joseartrivera As the test data is discussed in another issue thread, maybe we can do it in another PR. Let's skip it for now?